packages/asli/asli.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Interpreter for Arm's Architecture Specification Language (ASL)"
+description: """
+Interpreter for the language ASL that Arm uses to write their ISA specifications.
+
+Currently supports reading/typechecking the specification,
+interactive execution of ASL statements and expressions,
+executing opcodes one at a time,
+loading ELF files and executing Arm binaries."""
+maintainer: "Alastair Reid <alastair.d.reid@gmail.com>"
+authors: "Alastair Reid"
+license: "BSD3"
+homepage: "https://github.com/alastairreid/asl-interpreter"
+bug-reports: "https://github.com/alastairreid/asl-interpreter/issues"
+depends: [
+  "ocaml" {>= "4.09"}
+  "linenoise"
+  "menhir"
+  "ocamlfind"
+  "ott"
+  "pprint"
+  "zarith"
+  "z3" {<= "4.7.1"}
+]
+build: [make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "asli"]
+install: [make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "install"]
+remove: [
+  make "INSTALL_DIR=%{prefix}%" "SHARE_DIR=%{asli:share}%" "uninstall"
+]
```

Comment:
`remove` field is unnecessary since opam 2.0. This should be removed

Reply:
Done.
Thanks for the help with this - I appreciate it.